operations/cancellationpolicies.md

```diff
@@ -0,0 +1,110 @@
+# Cancellation Policies
+
+## Get all cancellation policies
+
+<strong style="color:red">Note that this endpoint is currently in beta phase and breaking changes may still occur.</strong>
+
+Returns all cancellation policies filtered by services, rate groups and other filters. Note this operation uses [Pagination](../guidelines/pagination.md).
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/cancellationPolicies/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "ServiceIds": [
+        "e654f217-d1b5-46be-a820-e93ba568dfac"
+    ],
+    "CancellationPolicyIds": [
+        "fe795f96-0b64-445b-89ed-c032563f2bac"
+    ],
+    "RateGroupIds": [
+        "fe795f96-0b64-445b-89ed-c032563f2bac"
```

Comment:
This is is same as in `CancellationPolicyIds`

Put this value there, it's from response 👍 
```suggestion
        "deb9444e-6897-4f2a-86b4-aff100c2896e"
```